dwave/embedding/transforms.py

```diff
@@ -149,7 +152,9 @@ def embed_bqm(source_bqm, embedding, target_adjacency, chain_strength=1.0,
             target_bqm.add_variable(v, 0.0)
             continue
 
-        quadratic_chain_biases = chain_to_quadratic(chain, target_adjacency, chain_strength)
+        strength = chain_strength[idx] if chain_strength_is_seq else chain_strength
+
+        quadratic_chain_biases = chain_to_quadratic(chain, target_adjacency, strength)
```

Comment:
This will conflict with #321, it might be better to base your patch on https://github.com/boothby/dwave-system/tree/embedded_structure